### PR TITLE
Migration to Vite & Tests Fixed

### PR DESCRIPTION
**Pull request by JacobMGEvans**

### CODE REVIEW

File: firestore-debug.log [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Jul 10, 2021 3:27:56 PM com.google.cloud.datastore.emulator.firestore.websocket.WebSocketServer start
```
Comment (mpeyper) [inline firestore-debug.log:1]: This file should be removed and/or ignored?

File: ui-debug.log [hunk 2]
```diff
@@ -0,0 +1 @@
+Web / API server started at 127.0.0.1:4000
```
Comment (mpeyper) [inline ui-debug.log:1]: Another one to remove/ignore

File: vite.config.ts.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// vite.config.ts
```
Comment (mpeyper) [inline vite.config.ts.js:1]: Does this need to be excluded/ignored somewhere? 

Seems odd to have bother the TS and the JS checked in. Even outputting the JS for a config file seems off to me, but I'm too unfamiliar with cute setup to know if it's needed or not?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
```
Comment (mpeyper) [inline index.html:6]: Should this looks a bit more like the [old `public/index html` file](https://github.com/OpenSourceRaidGuild/website/blob/970b60a45aed7ea52e4883d9890fc3181bd2ed58/public/index.html)?

File: lint-staged.config.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
   '*.+(js|jsx|ts|tsx)': ['eslint --fix'],
-  '*.+(json|yml|yaml|css)': ['prettier --write'],
+  '*.+(json|yml|yaml|css|js|jsx|ts|tsx)': ['npx rome check --apply'],
```
Comment (nobrayner) [inline lint-staged.config.js:3]: Wait... So eslint and rome? 😛

File: vite.config.ts.js (hunk 3 above)
Comment (JacobMGEvans) [reply vite.config.ts.js:1]: Honestly no idea

**Review by mpeyper — APPROVED** [review]
Some comments left, but nothing major. Overall the changes make sense.

File: index.html (hunk 4 above)
Comment (JacobMGEvans) [reply index.html:6]: ahahaha Oops

File: ui-debug.log (hunk 2 above)
Comment (JacobMGEvans) [reply ui-debug.log]: added to gitignore

File: vite.config.ts.js (hunk 3 above)
Comment (JacobMGEvans) [reply vite.config.ts.js:1]: added to gitignore

File: index.html (hunk 4 above)
Comment (JacobMGEvans) [reply index.html]: Fixed Title "OSRG"

File: firestore-debug.log (hunk 1 above)
Comment (JacobMGEvans) [reply firestore-debug.log]: added to gitignore

**Review by mpeyper — APPROVED** [review]

File: vite.config.ts.js (hunk 3 above)
Comment (nobrayner) [reply vite.config.ts.js:1]: When I setup Vite, I never saw a generated file like this 🤔

File: types/static.d.ts [hunk 6]
```diff
@@ -1,59 +0,0 @@
-/* Use this file to declare any custom file extensions for importing */
```
Comment (nobrayner) [inline types/static.d.ts:1]: Isn't this generated by Vite? And provides TS support for these kinds of files?

Comment (JacobMGEvans) [reply types/static.d.ts:1]: No idea

File: lint-staged.config.js (hunk 5 above)
Comment (JacobMGEvans) [reply lint-staged.config.js:3]: oops

**Pull request merged**